jina/executors/indexers/query/compound.py

```diff
@@ -0,0 +1,28 @@
+from jina.executors.compound import CompoundExecutor
+from jina.executors.indexers.query import BaseQueryIndexer
+
+
+class QueryCompoundExecutor(CompoundExecutor, BaseQueryIndexer):
+    """A Compound Executor that wraps several QueryIndexers"""
+
+    # def get_query_handler(self):
+    #     """Get a *readable* index handler when the ``index_abspath`` already exist, need to be overridden"""
+    #     return 0
+    #
+    # def get_add_handler(self):
+    #     """Get a *writable* index handler when the ``index_abspath`` already exist, need to be overridden"""
+    #     return 0
+    #
+    # def get_create_handler(self):
+    #     """Get a *writable* index handler when the ``index_abspath`` does not exist, need to be overridden"""
+    #     return 0
```

Comment:
What should go in here? should this code be removed/uncommented or should some comment be added, that explains it?

Reply:
Removed.